### ISSUE TITLE
Freeze when adding page to Favorites

### ISSUE THREAD

**Issue by andyylin** [stevencohn/OneMore#162]
On version 3.14, I've found that OneNote freezes when I try to "Add current page" to Favorites. The entire OneNote window is no longer interactive, though Windows doesn't report the program as having frozen.

**Comment by stevencohn** [comment]
I haven't seen this happen and can't seem to reproduce it. Is this the first favorite you're trying to add? Do you have others and if so, how many?

**Comment by andyylin** [comment]
Yes, this is the first favorite I'm trying to add. I don't have any others.

Are there any log files that I can send to you to help figure out the reason?

**Comment by stevencohn** [comment]
Open the OneMore about box. That has a link to the log file.

**Comment by andyylin** [comment]
The log ([OneMore.log](https://github.com/stevencohn/OneMore/files/5863976/OneMore.log)) doesn't seem to indicate anything. It doesn't record a crash. The program hung between the two instances of OneNote opening in the log.

I just realized that it seems to be related to the Page Title having an emoji in it. I tried with another page and it worked fine. Is this a bug or a restriction? The problematic page title has the calendar emoji (📅) in it: 
![image](https://user-images.githubusercontent.com/51783311/105657868-45c35500-5f00-11eb-8c28-2fa7b264ee96.png)
